app/views/mainpages/index.html.erb

```diff
@@ -11,7 +11,7 @@
         <% end %>
         <div class='main_cupons_content'><%= render 'content'%></div>
       </div>
-      <div class='main_cupons_bar_right'><%= render 'category'%></div>
+      <div class='main_cupons_bar_right'><%= (render 'category') if current_user && current_user.email == "procuponss@gmail.com"%></div>
```

Comment:
I think that instead of the mail, you should put an environment variable.

```suggestion
      <div class='main_cupons_bar_right'><%= (render 'category') if current_user && current_user.email == EMAIL_ADMIN %></div>
```